regression-test/data/compaction/test_compaction_uniq_keys_row_store.out

```diff
@@ -47,3 +47,51 @@
 -- !point_select --
```

Comment:
please add none full prepared type sql cases, you could refer to this PR : https://github.com/apache/doris/pull/33807/files